Assets/Controllers/MouseInputManager.cs

```diff
@@ -7,18 +7,23 @@ public class MouseInputManager : MonoBehaviour {
 
     [SerializeField]
     int hexLayer;
+    [SerializeField]
+    float maxTimeAllowedToDoubleClick = 0.5f;
```

Comment:
So, a serialized field lets me see and adjust it in the editor.  I haven't decided how a double click 'should feel' yet, so I have it set here like this.  Once I've picked a value I like, I'll either make this a static, or I'll make that a default somewhere and have an option to change the time delay that allows a double click.